tests/Models/Casting.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Models;
+
+use MongoDB\Laravel\Eloquent\Model as Eloquent;
+
+class Casting extends Eloquent
+{
+    protected $connection = 'mongodb';
+    protected $collection = 'casting';
+
+    protected $fillable = [
```

Comment:
It seems more correct to have used `$fillable` instead of `$unguarded`. Could you merge `CastBinaryUuid` into the generic `Casting` model. I see no reason to separate them.

Reply:
There is also `CastObjectId` model that I think can be merged into `Casting`.

Reply:
There is specific behaviors with the `_id` field. It's safer to keep it separate.

Reply:
Done in 8d9736b.